ci/build-push-multiplatform.sh

```diff
@@ -82,33 +97,60 @@ function build_push() {
     esac
 
     local TAG
-    readonly TAG="${REPO_URL}:${BUILD_TAG}${BUILD_TYPE_SUFFIX}-${BUILD_ARCH}"
+    readonly TAG="${REPO_URL}:${BUILD_TAG}${BUILD_TYPE_SUFFIX}-${BUILD_PLATFORM}-${BUILD_ARCH}"
     local LATEST_TAG
-    readonly LATEST_TAG="${REPO_URL}:latest${BUILD_TYPE_SUFFIX}-${BUILD_ARCH}"
+    readonly LATEST_TAG="${REPO_URL}:latest${BUILD_TYPE_SUFFIX}-${BUILD_PLATFORM}-${BUILD_ARCH}"
 
     if [[ "${PUSH}" == true ]]; then
         echo "Building tags: ${TAG}, ${LATEST_TAG}"
-        docker buildx build \
-            --push \
-            --file "${DOCKERFILE}" \
-            --build-arg BUILD_TAG="${BUILD_TAG}" \
-            --build-arg BUILDKIT_INLINE_CACHE=1 \
-            --platform="${PLATFORM}" \
-            --tag "${LATEST_TAG}" \
-            --tag "${TAG}" \
-            .
+
+        if [[ "${BUILD_PLATFORM}" == "windows" ]]; then
+            docker build \
+                --file "${DOCKERFILE}" \
+                --build-arg BUILD_TAG="${BUILD_TAG}" \
+                --build-arg BUILDKIT_INLINE_CACHE=1 \
+                --platform="${PLATFORM}" \
+                --tag "${LATEST_TAG}" \
+                .
+
+            docker tag "${LATEST_TAG}" "${TAG}"
+
+            docker push "${LATEST_TAG}"
+            docker push "${TAG}"
+        else
+            docker buildx build \
+                --push \
+                --file "${DOCKERFILE}" \
+                --build-arg BUILD_TAG="${BUILD_TAG}" \
+                --build-arg BUILDKIT_INLINE_CACHE=1 \
+                --platform="${PLATFORM}" \
+                --tag "${LATEST_TAG}" \
+                --tag "${TAG}" \
+                --provenance=false \
```

Comment:
`--provenance=false` has been added to prevent creating manifest list in place of manifest

Reply:
This should be a comment in the script.

Reply:
Done